### PR TITLE
Ac 1402 nodeservice downtime

### PR DESCRIPTION
**Pull request by JSLKM**
#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [x] the documentation is updated or intregrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: lib/launch-fastify.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const importEnv = require('./import-env')
 const { exportFastifyOptions, exportServiceOptions } = require('./options-extractors')
 const statusRoutes = require('./status-routes')
 const swaggerDefinition = require('./swagger-definition')
+const WAIT_BEFORE_SERVER_CLOSE = parseInt(process.env.WAIT_BEFORE_SERVER_CLOSE) || 10
```
Comment (allevo) [inline lib/launch-fastify.js]: WAIT_BEFORE_SERVER_CLOSE => WAIT_BEFORE_SERVER_CLOSE_SEC

File: lib/launch-fastify.js [hunk 2]
```diff
@@ -104,6 +105,17 @@ async function launchFastify(file, options, address) {
...
+    // Google Kubernetes Engine (GKE) is 10 sec,
+    // so the worst case, the iptables rule will
+    // be updated 10 seconds later after the pod
+    // deletion event arrives
+
+    setTimeout(() => {
+      fastify.close()
+    }, WAIT_BEFORE_SERVER_CLOSE * 1000)
+  })
+
```
Comment (allevo) [inline lib/launch-fastify.js]: Move into `launch` function

File: lib/launch-fastify.js [hunk 3]
```diff
@@ -104,6 +105,17 @@ async function launchFastify(file, options, address) {
     await fastify.listen(options.port, address)
   }
 
+  process.on('SIGTERM', () => {
+    // Google Kubernetes Engine (GKE) is 10 sec,
+    // so the worst case, the iptables rule will
+    // be updated 10 seconds later after the pod
+    // deletion event arrives
```
Comment (allevo) [inline lib/launch-fastify.js]: link?

File: tests/modules/immediate-close-module.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Mia srl
```
Comment (JGiola) [inline tests/modules/immediate-close-module.js]:
```suggestion
 * Copyright 2020 Mia srl
```

**Pull request merged**